ts/pages/api.tsx

```diff
@@ -0,0 +1,830 @@
+                                <RequestConfigurationDetailLabel>sellToken</RequestConfigurationDetailLabel>
+                                <RequestConfigurationDetailValue>USDC</RequestConfigurationDetailValue>
+                            </RequestConfigurationDetail>
+                        </RequestConfigurationRow>
+                    </RequestContainer>
+                    <ResponseContainer>
+                        <SyntaxHighlighter
+                            language={'json'}
+                            style={customJsonSyntaxStyle}
+                            PreTag={FadedJSONResponsePre}
+                        >
+                            {quote ? JSON.stringify(quote, null, 2) : ''}
+                        </SyntaxHighlighter>
+                    </ResponseContainer>
+                </FlexColumnWrapped>
+            </Section>
+            {/* JavaScript Code Sample Section */}
+            <Section bgColor="dark" isFlex={true} maxWidth="1170px">
+                <VerticalFlow>
+                    <SmallerTitle>Swap tokens in any product experience</SmallerTitle>
+                    <CenteredDescription>
+                        Call the API to get a quote, pass the order through your smart contracts,
+                        <br />
+                        and execute the trade at the best price possible.
+                    </CenteredDescription>
+                    <div style={{ width: '100%', maxWidth: 1040 }}>
+                        <SwapCodeExampleContainer>
+                            <SwapCodeExampleTitleRow>
+                                <SwapCodeLabel>Fill order in JavaScript</SwapCodeLabel>
+                                <SwapCodeExampleLaunchExampleContainer>
+                                    <Button
+                                        fontSize={'16px'}
+                                        target={'_blank'}
+                                        href="https://codepen.io/fragosti/pen/xxbmgqy"
+                                        isWithArrow={true}
+                                        isAccentColor={true}
+                                    >
+                                        Launch in CodePen
+                                    </Button>
+                                </SwapCodeExampleLaunchExampleContainer>
+                            </SwapCodeExampleTitleRow>
+                            <SyntaxHighlighterContainer>
+                                <SyntaxHighlighter
+                                    style={dark}
+                                    PreTag={TypescriptCodeSamplePre}
+                                    showLineNumbers={true}
+                                    language={'typescript'}
+                                >
+                                    {`// Get a quote to sell 1 ETH to buy DAI
+const response = await fetch(
+  'https://api.0x.org/swap/v0/quote?sellToken=ETH&buyToken=DAI&sellAmount=1000000000000000000'
```

Comment:
Feel free to ignore this but 1 ETH might be a bit much if people try copy pasting this and accept the tx without realizing that they are making a live trade 😄

Reply:
Agreed - going to touch this up today